### DISCUSSION TITLE
Mesh development and community support

### DISCUSSION THREAD

**Discussion by saeed-moghimi-noaa** [oceanmodeling/Management#19] [General]
@gseroka @josephzhang8 @SorooshMani-NOAA @feiye-vims  @yangz888 @brey 

All,

We as a team reach to the point that we need to think about how to share current SCHISM meshes with the community so they can go ahead and subset those for their own local domain without or with limited support from Dev team. 

Therefore I am asking you on what, 
1. how and where we can share our current meshes for:
- Atlantic Domain (operational)
- Pacific Domain (pre-operational)
- Global Domain (VIMS)

2. We also need to provide some simple Python codes or tutorials on how to:
- Subset mesh
- Generate mesh from scratch
- Generate run folder
- Download some data 
- Perform basic model data inter-comparison perhaps for tidal case

@SorooshMani-NOAA  and Team at NOAA is sharing some routines here:
https://github.com/SorooshMani-NOAA/OCSMeshTutorial

Please share your view.

Thanks,
-Saeed

**Comment by feiye-vims** [comment]
For 2.2 Generate mesh from scratch, we have a tutorial at https://schism-dev.github.io/schism/master/mesh-generation/meshing-for-compound-floods/overview.html, but it is still unfinished.

**Comment by josephzhang8** [comment]
Global SCHISM 3D mesh (and model setup) has been uploaded to zenodo as a requirement for the GMD paper:

https://zenodo.org/record/7779837
